prediction_market_agent/agents/microchain_agent/tools.py

```diff
@@ -0,0 +1,53 @@
+from eth_typing import HexStr, HexAddress, ChecksumAddress
```

Comment:
just as a heads up, I moved this to microchain_agent/utils.py. I thought 'tools' might be confusing because 'tools/functions' have a loaded meaning in the context of agents (i.e. some code that is registered with the agent, and it can call).